webapp/channels/src/components/autocomplete_selector.tsx

```diff
@@ -96,7 +109,13 @@ export default class AutocompleteSelector extends React.PureComponent {
             disabled,
             listComponent,
             listPosition,
-        } = this.props;
```

Comment:
There is no need for this in order to generate the default values. Instead of doing:
```
const {foo, faa} = {foo = 1, ...props}
```
You can do:
```
const {foo = 1, faa} = props;
```

Reply:
Thank you, I didn't know it works like that. Because I added the `defaultProps` back, the defaults are removed.